doc/quick_start.md

```diff
@@ -71,17 +78,29 @@ tools for an isolated Python environment.
 
 Use the following command to install `flink-ml-framwork`
 ```bash
+# Install latest stable version
 pip install flink-ml-framework
+
+# Install latest snapshot version
+pip install --pre flink-ml-framework
```

Comment:
I followed the instructions in the QuickStart but was not able to run the example successfully. Those instructions also seem a bit too complicated for a QuickStart.

Could you check QuickStart doc from other popular open source projects and check if our QuickStart doc is as easy to use as theirs?

And could you verify the commands specified in this doc to make sure users can easily follow (e.g. mostly copy&paste) the given instructions to complete the quick start? 

And let's go over the README and the QuickStart tomorrow.

Reply:
It is updated according to our discussion at e87d171